main/settings.py

```diff
@@ -1020,7 +1020,9 @@ def get_all_config_keys():
 
 # Relative URL to be used by Djoser for the link in the password reset email
 # (see: http://djoser.readthedocs.io/en/stable/settings.html#password-reset-confirm-url)
-PASSWORD_RESET_CONFIRM_URL = "password_reset/confirm/{uid}/{token}/"
+# fmt: off
+PASSWORD_RESET_CONFIRM_URL = "password_reset/confirm/{uid}/{token}/"  # pragma: allowlist secret
+# fmt: on
```

Comment:
Can you cross-match the same setting with xPRO, We didn't to turn the fmt off/on there. Instead, we used 
```
PASSWORD_RESET_CONFIRM_URL = "password_reset/confirm/{uid}/{token}/"  # noqa: S105 # pragma: allowlist secret
```